### DISCUSSION TITLE
Generate automatic limits in chart. - .Net Maui

### DISCUSSION THREAD

**Discussion by math242j** [dotnet/maui#14013] [Q&A]
Hey!
Is it possible to set a default NummericalAxis on the YAxes so the chart will be more usefull.
I think something like this: 
"<chart:NumericalAxis Maximum="({Data Binding}+2)" Minimum="({Data Binding} -2)" "

This my code now, but the the YAxes nummer, is not dynamic:
                      <chart:SfCartesianChart.YAxes>
                            <chart:NumericalAxis Maximum="40" Minimum="36" Interval="1">
                                <chart:NumericalAxis.Title>
                                    <chart:ChartAxisTitle Text="Kg. EKM pr. ko"></chart:ChartAxisTitle>
                                </chart:NumericalAxis.Title>
                            </chart:NumericalAxis>
                        </chart:SfCartesianChart.YAxes>
                    </chart:SfCartesianChart>

**Comment by hartez** [comment]
I think you might have better luck on the [SyncFusion forums](https://www.syncfusion.com/forums/maui?control=sfcartesianchart).
